filebeat/filebeat.full.yml

```diff
@@ -395,22 +395,206 @@ output.elasticsearch:
+  # are disabled. The default is 0 seconds.
+  #keep_alive: 0
+
+  # Sets the output compression codec. Must be one of none, snappy and gzip. The
+  # default is snappy.
+  #compression: snappy
+
+  # The maximum permitted size of JSON-encoded messages. Bigger messages will be
+  # dropped. The default value is 1000000 (bytes). This value should be equal to
+  # or less than the broker’s message.max.bytes.
+  #max_message_bytes: 1000000
+
+  # The ACK reliability level required from broker. 0=no response, 1=wait for
+  # local commit, -1=wait for all replicas to commit. The default is 1.  Note:
+  # If set to 0, no ACKs are returned by Kafka. Messages might be lost silently
+  # on error.
+  #required_acks: 0
+
+  # The number of seconds to wait for new events between two producer API calls.
+  #flush_interval: 1
+
+  # The configurable ClientID used for logging, debugging, and auditing
+  # purposes.  The default is "beats".
+  #client_id: beats
+
+  # Optional TLS. By default is off.
+  # List of root certificates for HTTPS server verifications
+  #tls.certificate_authorities: ["/etc/pki/root/ca.pem"]
+
+  # Certificate for TLS client authentication
+  #tls.certificate: "/etc/pki/client/cert.pem"
+
+  # Client Certificate Key
+  #tls.certificate_key: "/etc/pki/client/cert.key"
+
+  # Controls whether the client verifies server certificates and host name.
+  # If insecure is set to true, all server host names and certificates will be
+  # accepted. In this mode TLS based connections are susceptible to
+  # man-in-the-middle attacks. Use only for testing.
+  #tls.insecure: true
+
+  # Configure cipher suites to be used for TLS connections
+  #tls.cipher_suites: []
+
+  # Configure curve types for ECDHE based cipher suites
+  #tls.curve_types: []
+
+#------------------------------- Redis output ---------------------------------
+#output.redis:
+  # The list of Redis servers to connect to. If load balancing is enabled, the
+  # events are distributed to the servers in the list. If one server becomes
+  # unreachable, the events are distributed to the reachable servers only.
+  #hosts: ["localhost:6379"]
+
+  # The Redis port to use if hosts does not contain a port number. The default
+  # is 6379.
+  #port: 6379
+
+  # The name of the Redis list or channel the events are published to. The
+  # default is filebeat.
+  #index: filebeat
+
+  # The password to authenticate with. The default is no authentication.
+  #password:
+
+  # The Redis database number where the events are published. The default is 0.
+  #db: 0
+
+  # The Redis data type to use for publishing events. If the data type is list,
+  # the Redis RPUSH command is used. If the data type is channel, the Redis
+  # PUBLISH command is used. The default value is list.
+  #datetype: list
+
+  # The Redis host to connect to when using topology map support. Topology map
+  # support is disabled if this option is not set.
+  #host_topology:
```

Comment:
Seems like topology handling is still a feature of the Redis output. We could consider deprecating it or marking it experimental or something like that.